lib/App/Sqitch/Command/rebase.pm

```diff
@@ -27,10 +27,17 @@ has upto_change => (
     isa => Str,
 );
 
+has revised => (
+    is  => 'ro',
+    isa => Bool,
+    default => 0,
+);
+
 sub options {
     return qw(
         onto-change|onto=s
         upto-change|upto=s
+        revised!
```

Comment:
I don't think we want this to be negate-able; the `!` means that there is also a `--no-revised` option, which doesn't seem necessary. Am I missing something?